python/tvm/relax/frontend/nn/core.py

```diff
@@ -360,12 +360,31 @@ def to(self, dtype: Optional[str] = None) -> None:  # pylint: disable=invalid-na
     def export_tvm(
         self,
         spec: "_spec.ModuleSpecType",
+        debug: bool = False,
```

Comment:
Shall we also add this `debug` flag to the `jit` method?

Reply:
Oh actually you will need the `debug` flag in line 405

Reply:
Good point, now added.